core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/tenant/JDBCTenantManager.java

```diff
@@ -719,12 +759,17 @@ public Tenant getTenant(String tenantUniqueID) throws UserStoreException {
         Tenant tenant = null;
         int id;
         try {
+
```

Comment:
Unnecessary new line

Reply:
fixed in [c5d56a3](https://github.com/wso2/carbon-kernel/pull/3316/commits/c5d56a3a83eca6381d1c85f1f3f0cb515b490c76)

Reply:
resolved in [c5d56a3](https://github.com/wso2/carbon-kernel/pull/3316/commits/c5d56a3a83eca6381d1c85f1f3f0cb515b490c76)